core/src/trezor/ui/layouts/common.py

```diff
@@ -64,7 +64,7 @@ def raise_if_not_confirmed(
     exc: ExceptionType = ActionCancelled,
 ) -> Awaitable[None]:
     action = interact(layout_obj, br_name, br_code, exc)
-    return action  # type: ignore ["UiResult" is incompatible with "None"]
+    return action  # type: ignore [Expression of type "Coroutine[Any, Any, UiResult]" is incompatible with return type "Awaitable[None]"]
```

Comment:
Was pyright unhappy about this, and why didn't CI catch it?

Reply:
I was having some problems locally, but CI is (probably) happy with both versions. Do you want to have the old version back  (i.e. do not fix it if it is not broken) or is it ok as I have it in this PR?

Reply:
Returned back to the previous state (as in `main`)